src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -35,6 +35,8 @@ contract SecurityCouncilManager is Initializable, ArbitrumTimelock, ISecurityCou
 
     uint256 public rotationNonce = 0;
 
+    uint256 public updateNonce = 0;
```

Comment:
https://github.com/ArbitrumFoundation/governance/pull/53#discussion_r1239974438
thinking also send this nonce to L1 and on L1 ensure it can only execute the next nonce

Reply:
oh i see. might make more sense to just use/enforce the "predecessor"?